test/performance-benchmark/util_visualization/visualization_time.py

```diff
@@ -0,0 +1,197 @@
+  "cvmfs.n_fs_inode_replace": "FS: #Stale inodes replaced by up-to-date version",
+  "cvmfs.n_fs_lookup": "FS: #Lookups",
+  "cvmfs.n_fs_lookup_negative": "FS: #Negative lookups",
+  "cvmfs.n_fs_open": "FS: #Files opened",
+  "cvmfs.n_fs_read": "FS: #Files read",
+  "cvmfs.n_fs_readlink": "FS: #Links read",
+  "cvmfs.n_fs_stat": "FS: #Stats",
+  "cvmfs.n_fs_stat_stale": "FS: #Stats for stale regular files",
+  "cvmfs.n_fs_statfs": "FS: #Statsfs calls",
+  "cvmfs.n_fs_statfs_cached": "FS: #Statsfs calls to cached statfs",
+  "cvmfs.n_io_error": "#I/O errors",
+  "cvmfs.no_open_dirs": "#Currently opened directories",
+  "cvmfs.no_open_files": "#Currently opened files",
+  "dentry_tracker.n_insert": "Dentry: #Added negative cache entries",
+  "dentry_tracker.n_prune": "Dentry: #Prune calls",
+  "dentry_tracker.n_remove": "Dentry: #Evicted negative cache entries",
+  "download-external.n_host_failover": "External dwldmgr: #Host failovers",
+  "download-external.n_proxy_failover": "External dwldmgr: #Proxy failovers",
+  "download-external.n_requests": "External dwldmgr: #Requests",
+  "download-external.n_retries": "External dwldmgr: #Retries",
+  "download-external.sz_transfer_time": "External dwldmgr: Transfer time (msec)",
+  "download-external.sz_transferred_bytes": "External dwldmgr: #Transferred bytes",
+  "download.n_host_failover": "Dwldmgr: #Host failovers",
+  "download.n_proxy_failover": "Dwldmgr: #Proxy failovers",
+  "download.n_requests": "Dwldmgr: #Requests",
+  "download.n_retries": "Dwldmgr: #Retries",
+  "download.sz_transfer_time": "Dwldmgr: Transfer time (msec)",
+  "download.sz_transferred_bytes": "Dwldmgr: #Transferred bytes",
+  "eio.01": "#EIO in cvmfs.cc:cvmfs_lookup()",
+  "eio.02": "#EIO in cvmfs.cc:ReplyNegative()",
+  "eio.03": "#EIO in cvmfs.cc:cvmfs_opendir()",
+  "eio.04": "#EIO in cvmfs.cc:cvmfs_open()",
+  "eio.05": "#EIO in cvmfs.cc:cvmfs_read()",
+  "eio.06": "#EIO in cvmfs.cc:cvmfs_open()",
+  "eio.07": "#EIO in cvmfs.cc:cvmfs_read()",
+  "eio.08": "#EIO in cvmfs.cc:cvmfs_read()",
+  "eio.total": "EIO: Sum of individual eio counters",
+  "fetch-external.n_downloads": "External fetch: #Downloaded files",
+  "fetch-external.n_invocations": "External fetch: #Object requests",
+  "fetch.n_downloads": "Fetch: #Downloaded files",
+  "fetch.n_invocations": "Fetch: #Object requests",
+  "inode_cache.n_drop": "Inode cache: #Drops",
+  "inode_cache.n_forget": "Inode cache: #Forgets",
+  "inode_cache.n_hit": "Inode cache: #Hits",
+  "inode_cache.n_insert": "Inode cache: #Inserts",
+  "inode_cache.n_insert_negative": "Inode cache: #Negative inserts",
+  "inode_cache.n_miss": "Inode cache: #Misses",
+  "inode_cache.n_replace": "Inode cache: #Replaces",
+  "inode_cache.n_update": "Inode cache: #Updates",
+  "inode_cache.n_update_value": "Inode cache: #Value changes",
+  "inode_cache.sz_allocated": "Inode cache: #Allocated bytes",
+  "inode_cache.sz_size": "Inode cache: Total size", # TODO which size? bytes?
```

Comment:
Yes, I believe this is given in bytes. Same below.

Reply:
done